Changed the 'key' input value of the 'Get full detection details' task (#27722)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_27.md

#### Playbooks

##### CrowdStrike Falcon Malware - Verify Containment Actions

> - Fixed an issue with the *'key'* input of the ***'Set'*** automation used within the **'Get full detection details'** task. 

Reviewer: ```suggestion
Fixed an issue with the *'Get full detection details'* task's key.
```

Author: done
